requirements.txt

```diff
@@ -53,4 +53,4 @@ urllib3==1.25.8
 vine==1.3.0
 webencodings==0.5.1
 Werkzeug==1.0.0
-zipp==3.1.0
+zipp==1.2.0
```

Comment:
Surprised to zipp bump so low. I guess as long as the tests pass this is fine though

Reply:
Yes, I was quite surprised by that, but it's a sub-dependency of importlib_metadata which we don't use so I figured it was fine